From 2e7d7aa03525954de2d5e9b352732d788e2ad024 Mon Sep 17 00:00:00 2001
From: Drew Moseley <drew.moseley@mender.io>
Date: Wed, 31 May 2017 17:50:59 -0400
Subject: [PATCH 6/7] u-boot-fslc: Set default environment variables for PXE
 usage

Signed-off-by: Drew Moseley <drew.moseley@mender.io>
---
 include/configs/udoo_neo.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/udoo_neo.h b/include/configs/udoo_neo.h
index 1d737cc..7fd9422 100644
--- a/include/configs/udoo_neo.h
+++ b/include/configs/udoo_neo.h
@@ -68,6 +68,7 @@
 
 /* Miscellaneous configurable options */
 #define CONFIG_SYS_MEMTEST_START	0x80000000
+#define CONFIG_LOADADDR			0x82000000
 #define CONFIG_SYS_MEMTEST_END		(CONFIG_SYS_MEMTEST_START + 0x10000)
 #define CONFIG_STACKSIZE		SZ_128K
 
-- 
2.7.4

