Index: git/boards.cfg
===================================================================
--- git.orig/boards.cfg
+++ git/boards.cfg
@@ -304,8 +304,8 @@ Active  arm         armv7          mx5
 Active  arm         armv7          mx5         genesi          mx51_efikamx        mx51_efikamx                         mx51_efikamx:MACH_TYPE=MACH_TYPE_MX51_EFIKAMX,IMX_CONFIG=board/genesi/mx51_efikamx/imximage_mx.cfg                                -
 Active  arm         armv7          mx5         genesi          mx51_efikamx        mx51_efikasb                         mx51_efikamx:MACH_TYPE=MACH_TYPE_MX51_EFIKASB,IMX_CONFIG=board/genesi/mx51_efikamx/imximage_sb.cfg                                -
 Active  arm         armv7          mx5         ttcontrol       vision2             vision2                              vision2:IMX_CONFIG=board/ttcontrol/vision2/imximage_hynix.cfg                                                                     Stefano Babic <sbabic@denx.de>
-Active  arm         armv7          mx6         udoo            quad_dual           udoo_dual                            udoo:IMX_CONFIG=board/udoo/quad_dual/udoo.cfg,MX6DL,DDR_MB=1024                                                                    Fabio Estevam <fabio.estevam@freescale.com>
-Active  arm         armv7          mx6         udoo            quad_dual           udoo_quad                            udoo:IMX_CONFIG=board/udoo/quad_dual/udoo.cfg,MX6Q,DDR_MB=1024                                                                    Fabio Estevam <fabio.estevam@freescale.com>
+Active  arm         armv7          mx6         udoo            quad_dual           udoo_dual                            udoo:IMX_CONFIG=board/udoo/quad_dual/udoo.cfg,MX6DL,DEFAULT_FDT_FILE="dts/imx6dl-udoo.dtb",DDR_MB=1024                                                                    Fabio Estevam <fabio.estevam@freescale.com>
+Active  arm         armv7          mx6         udoo            quad_dual           udoo_quad                            udoo:IMX_CONFIG=board/udoo/quad_dual/udoo.cfg,MX6Q,DEFAULT_FDT_FILE="dts/imx6q-udoo.dtb",DDR_MB=1024                                                                    Fabio Estevam <fabio.estevam@freescale.com>
 Active  arm         armv7          mx6         udoo            neo                 udoo_neo                             udoo_neo:IMX_CONFIG=board/udoo/neo/udoo_neo.cfg,MX6SX,DEFAULT_FDT_FILE="imx6sx-udoo-neo.dtb",DDR_MB=1024                          Jasbir Matharu <jasbirm66@gmail.com>
 Active  arm         armv7          mx6         udoo            neo                 udoo_neo_hdmi                        udoo_neo:IMX_CONFIG=board/udoo/neo/udoo_neo.cfg,MX6SX,VIDEO,UDOO_NEO_HDMI,VIDEO_PANEL="MCIMX28LCD",DEFAULT_FDT_FILE="imx6sx-udoo-neo.dtb",DDR_MB=1024                   Jasbir Matharu <jasbirm66@gmail.com>
 Active  arm         armv7          mx6         udoo            neo                 udoo_neo_m4                          udoo_neo:IMX_CONFIG=board/udoo/neo/udoo_neo.cfg,MX6SX,DEFAULT_FDT_FILE="imx6sx-udoo-neo.dtb",DDR_MB=1024                       Jasbir Matharu <jasbirm66@gmail.com>
Index: git/include/configs/udoo.h
===================================================================
--- git.orig/include/configs/udoo.h
+++ git/include/configs/udoo.h
@@ -96,12 +96,6 @@
 #define CONFIG_CMD_FAT
 #define CONFIG_DOS_PARTITION
 
-#if defined(CONFIG_MX6Q)
-#define CONFIG_DEFAULT_FDT_FILE		"dts/imx6q-udoo.dtb"
-#else
-#define CONFIG_DEFAULT_FDT_FILE		"dts/imx6dl-udoo.dtb"
-#endif
-
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"script=uEnv.txt\0" \
 	"image=zImage\0" \
