Index: kernel-source/Makefile
===================================================================
--- kernel-source.orig/Makefile
+++ kernel-source/Makefile
@@ -554,7 +554,9 @@ $(KCONFIG_CONFIG) include/config/auto.co
 # if auto.conf.cmd is missing then we are probably in a cleaned tree so
 # we execute the config step to be sure to catch updated Kconfig files
 include/config/%.conf: $(KCONFIG_CONFIG) include/config/auto.conf.cmd
-	$(Q)$(MAKE) -f $(srctree)/Makefile silentoldconfig
+	$(Q)if [ ! -e include/config/auto.conf ]; then \
+                $(MAKE) -f $(srctree)/Makefile silentoldconfig ;\
+            fi;
 else
 # external modules needs include/generated/autoconf.h and include/config/auto.conf
 # but do not care if they are up-to-date. Use auto.conf to trigger the test
